cedar-policy/src/api.rs

```diff
@@ -867,29 +872,24 @@ impl From<cedar_policy_validator::SchemaError> for SchemaError {
             cedar_policy_validator::SchemaError::CycleInActionHierarchy => {
                 Self::CycleInActionHierarchy
             }
-            cedar_policy_validator::SchemaError::EntityTypeParseError(e) => {
-                Self::EntityTypeParse(e)
-            }
-            cedar_policy_validator::SchemaError::NamespaceParseError(e) => Self::NamespaceParse(e),
-            cedar_policy_validator::SchemaError::CommonTypeParseError(e) => {
+            cedar_policy_validator::SchemaError::ParseEntityType(e) => Self::EntityTypeParse(e),
+            cedar_policy_validator::SchemaError::ParseNamespace(e) => Self::NamespaceParse(e),
```

Comment:
Looks a little weird here that the names are reversed for the `cedar-policy` errors vs. the `cedar-policy-validator` errors. I prefer the names with `Parse` first.

Reply:
This PR is restricted to `cedar_policy_validator`. I'll update the one in the Cedar APIs in #231.